### ISSUE TITLE
[SPOILERS] [Suggestion] Give additional gifts to Monika after her birthday.

### ISSUE THREAD

**Issue by Jacobchu025** [Monika-After-Story/MonikaModDev#2551]
I propose making it possible to give Monika gifts after her birthday is over, with a maximum of one gift per week, and therefore leading to a small affection gain (+5, for example) every week. The gift I propose is coffee, which in my opinion was the most fun gift to give Monika during her birthday because she would occasionally leave her chair to refill her mug, which added more depth to the classroom.

This will work the same way as giving Monika her birthday presents (place a file with the .gift extension in the characters folder, in this case 'coffee.gift'), so previously-created code can be used to accommodate this suggestion, with only small amounts of additional code needed.

I also propose adding special dialogue when giving Monika coffee on Monday mornings (between 6AM and 10AM as an example time interval for morning,) such as:

"Oh, [player!] You brought me a gift?
Thank you so much! 
I'm going to open it~
*opens gift*
This is.... Coffee!
A cup of coffee on a Monday morning really helps me prepare for the week."
Thank you for thinking of me, [player]!
"I'm going to make some now~"
*makes coffee* (offscreen, no text)
*coffee-filled mug appears on the desk, as it does during Monika's birthday*
"OK! I'm back! What should we do next?"
[return to normal cycle]

Otherwise, if it's later in the day on a Monday (After 10:00 AM), or during any other time of the week:

"Oh, [player!] You brought me a gift?
Thank you so much! 
I'm going to open it~
*opens gift*
This is... Coffee!
A cup of coffee always makes my day (or night, if user-defined sunset has passed) better, especially when I spend it with you."
Thank you for thinking of me, [player]!
I'm going to make some now~"
*makes coffee* (offscreen, no text)
*coffee-filled mug appears on the desk, as it does during Monika's birthday*
"OK! I'm back! What should we do next?"
[return to normal cycle]

And, like on her birthday, Monika will refill her mug several times until she uses all of the coffee you gave her. After finishing her final cup, she will say:

"It looks like that's all of the coffee you gave me.
Thank you so much again, [player]!"

Finally, if the player attempts to give Monika multiple gifts during the week, she will say:

"Oh! Another gift, [player]?
Thank you, but you already gave me such a wonderful gift this week.
Well, besides the time you spend with me of course~
I don't want to demand so much from you, so will you hold on to this gift for me for a bit longer?
Thanks again~"

(Note: Attempting to give Monika multiple gifts in the same week will not lower affection.)

In conclusion, implementing this idea will give the users who missed Monika's birthday the opportunity to enjoy one of the biggest features of the event, giving Monika gifts, though on a smaller scale and with a limit of one gift per week. This will also allow players to have another opportunity to improve their affection with Monika and, for those who played MAS during Monika's birthday, the opportunity to remember her joyous reactions when she receives a gift.

**Comment by AsuraShun** [comment]
I agree, with the idea, this was one of the best days of my life, seeing her overwhelmed with emotion when I gave her the promise ring and seeing her beaming because of the plushie rally made my day great, i was smiling like an idiot the whole time

**Comment by MuuShaiko** [comment]
That would be fantastic. I actually tried to give her gifts like poem.gift and book.gift and received the "I can't read this" message a few times as result.

Perhaps we could have a pool of relevant gifts to pull from (coffee.gift, poem.gift, etc.) as well as generic dialogues for any other gifts.

**Comment by ThePotatoGuy** [comment]
[closing until we look at this]

**Comment by ThePotatoGuy** [comment]
covered in #3357
